cleanlab/internal/util.py

```diff
@@ -688,6 +688,12 @@ def num_unique_classes(labels, multi_label=None) -> int:
         return len(set(labels))
 
 
+def _binarize_pred_probs_slice(pred_probs: np.ndarray, class_num: int) -> np.ndarray:
```

Comment:
would be nice to add 1-line descriptive comment, eg:
""" This function does XYZ... "

Reply:
Fixed: https://github.com/aditya1503/cleanlab/blob/940558449f0dd07d69b775bc20dfbeb41fa311c6/cleanlab/internal/util.py#L692